frontend/src/pages/AccountReceive.vue

```diff
@@ -11,7 +11,19 @@
       </div>
     </div>
 
-    <div v-else class="q-mx-auto" style="max-width: 800px">
+    <div
+      v-else-if="!isAccountSetup && !advancedMode"
+      class="dark-toggle text-center text-bold q-pa-md q-mb-lg"
```

Comment:
IMO we should keep the max-width, it looks odd to have the message take the full width of the page

<img width="753" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/17163988/a1426b17-ebb3-42c1-8fe4-7561cd5022ee">

Reply:
Addressed it in https://github.com/ScopeLift/umbra-protocol/commit/d34d75c3e854848b58237035fb4eea4868304fdc